ow_dynamic_terrain/src/TerrainModifier.cpp

```diff
@@ -243,29 +243,31 @@ bool TerrainModifier::applyImageToHeightmap(Heightmap* heightmap, const cv::Poin
     return false;
   }
 
-  auto heightmap_size = static_cast<int>(terrain->getSize());
-  auto left = max(center.x - image.cols / 2, 0);
-  auto top = max(center.y - image.rows / 2, 0);
-  auto right = min(center.x - image.cols / 2 + image.cols - 1, heightmap_size);
-  auto bottom = min(center.y - image.rows / 2 + image.rows - 1, heightmap_size);
+  int heightmap_size = static_cast<int>(terrain->getSize());
```

Comment:
heightmap_size is not used and so it seems this line can be deleted.

Reply:
Thanks, fixed. I meant to remove that and messed it up somehow.